nightingale_body/src/nightingale_body/body_control.py

```diff
@@ -1,21 +1,25 @@
 #!/usr/bin/env python3
 
 import actionlib
+import numpy as np
 import rospy
 
```

Comment:
import nightingale_manipulation.manipulation_action_client.joint_goal instead of redefining it with the _make_goal function. I could also move joint_goal into a nightingale_manipulation.utils file if that would look better

Reply:
didnt mean to specifically highlight this line here sorry, it wouldn't let me highlight the _make_goal function definition for some reason.

Reply:
I agree that the goal definition function shouldn't really be repeated here, but I'm not sure adding this dependency is wise since broke out the body control from the manipulation to remove any dependencies between the two.